addons/udes_stock/models/stock_move_line.py

```diff
@@ -652,6 +651,8 @@ def _mark_as_done(self, values, split=True):
             )
 
         if split:
+            if values.get("location_dest_id"):
+                self.location_dest_id = values["location_dest_id"]
```

Comment:
Isn't this trying to set `location_dest_id` object with integer id value?

Reply:
Yes when you write a many2one field it handles both integer or recordset